notebooks/human-SSID-query.ipynb

```diff
@@ -0,0 +1,143 @@
+     ]
+    }
+   ],
+   "source": [
+    "print (output)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "As you can see, the output result provides numerous types of information related to our queried variant  \n",
+    "We now need to sieve through the output to parse out the information we want"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "chrom = output['_embedded']['variantList'][0]['chromosome']\n",
+    "locat = output['_embedded']['variantList'][0]['start']\n",
+    "ref = output['_embedded']['variantList'][0]['reference']\n",
+    "alt = output['_embedded']['variantList'][0]['alternate']\n",
+    "cluster = output['_embedded']['variantList'][0]['ids']"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The variant location, reference & alternate alleles, as well as the SS IDs have been set to 5 variables (*chrom, locat, ref, alt, clustered*). We can output the result of our query below:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Variant_location = 19:45411941\n",
+      "Reference/Alternate = T/C\n",
+      "SS_IDs = [u'ss1578653635', u'ss782605255', u'ss661874596', u'ss228156553', u'ss1067590770', u'ss80743998', u'ss76884559', u'ss481897219', u'ss491161751', u'ss159740185', u'ss484318107', u'ss713519649', u'ss491549548', u'ss160670648', u'ss132769779', u'ss244239676', u'ss1681037690', u'ss44158325', u'ss870163', u'ss168243995', u'ss537712916', u'ss1693665397', u'ss994339187', u'ss96308980', u'ss974506672', u'ss836101974', u'ss536501696', u'ss410828263', u'ss24811489', u'ss1363326184', u'ss1698394130', u'ss1638043657', u'ss569295', u'ss181341884', u'ss803061', u'ss275517967', u'ss469414605', u'ss2419938', u'ss21518782', u'ss237689596', u'rs429358', u'ss12568607', u'ss483834108', u'ss16231123', u'ss107936537', u'ss780608662']\n"
+     ]
+    }
+   ],
+   "source": [
+    "print ('Variant_location = ', (chrom), \":\", (locat), sep='')\n",
+    "print ('Reference/Alternate = ', (ref), \"/\", (alt), sep='')\n",
+    "print ('SS_IDs = ', (cluster), sep='')"
```

Comment:
This is not only SS ids. It also contains RSids
It would actually be useful to have the rs and ss separated
```suggestion
    "print ('SS_IDs = ', (cluster), sep='')"
```